packages/cli/src/commands/mock.ts

```diff
@@ -57,15 +57,24 @@ const mockCommand: CommandModule = {
         },
       }),
   handler: parsedArgs => {
-    const { multiprocess, dynamic, port, host, cors, operations } = (parsedArgs as unknown) as CreatePrismOptions & {
+    const {
+      multiprocess,
+      dynamic,
+      port,
+      host,
+      cors,
+      operations,
+      spec,
+    } = (parsedArgs as unknown) as CreatePrismOptions & {
       multiprocess: boolean;
+      spec: string;
     };
 
     if (multiprocess) {
-      return createMultiProcessPrism({ cors, dynamic, port, host, operations });
+      return createMultiProcessPrism({ cors, dynamic, port, host, operations, spec });
```

Comment:
Ok I understand why you're sending the spec down here; I think we could probably move the chokidar part in this part of the file so we can simply call this function again?